### PR TITLE
Added two icons Request Send Phone and Split Cost.

### PR DESCRIPTION
**Pull request by vig9295**

### CODE REVIEW

File: app/stylesheets/SplitCost.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  width: 23px;
+  height: 25px;
+  fill: white;
+}
+.outline {
+  stroke: #CFD6DC;
+  stroke-width: 2px;
+}
+.bottom-fold {
+  fill: #CFD6DC;
```
Comment (BrettBukowski) [inline app/stylesheets/SplitCost.scss]: since this color is used several places now, do you think it makes sense to put into _colors.scss?

**Comment by vig9295** [comment]
ping

File: app/stylesheets/SplitCost.scss (hunk 1 above)
Comment (BrettBukowski) [inline app/stylesheets/SplitCost.scss]: ping - yeah, looks like this color is used in 9 different files now. Would you be willing to make a variable in _colors.scss

Comment (vig9295) [inline app/stylesheets/SplitCost.scss]: I can do that

**Comment by vig9295** [comment]
ping

**Comment by BrettBukowski** [comment]
lgtm

**Pull request merged**